tests/Functional/PortabilityTest.php

```diff
@@ -62,6 +62,20 @@ public function testCaseConversion(ColumnCase $case, array $expected): void
         self::assertSame($expected, array_keys($row));
     }
 
+    /** @param list<string> $expected */
+    #[DataProvider('caseProvider')]
+    public function testCaseConversionColumnName(ColumnCase $case, array $expected): void
+    {
+        $this->connectWithPortability(Connection::PORTABILITY_FIX_CASE, $case);
+        $this->createTable();
+
+        $result = $this->connection->executeQuery('SELECT * FROM portability_table');
+
+        foreach ($expected as $index => $name) {
+            self::assertSame($name, $result->getColumnName($index));
+        }
```

Comment:
Can we assert all columns at once instead of stopping at the first incorrect one?

Something like this, maybe?

```suggestion
        $actual = []
        foreach ($expected as $index => $name) {
            $actual[$index] = $result->getColumnName($index);
        }

        self::assertSame($expected, $actual);
```

Reply:
@derrabus I updated the test with your suggestion